recipes/spyne/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = "2.12.11" %}
+
+package:
+    name: spyne
+    version: {{ version }}
+
+source:
+    fn: spyne-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/s/spyne/spyne-{{ version }}.tar.gz
+    md5: 6073a472efa3e44647811a596b88cb4a
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    entry_points:
+        - sort_wsdl=spyne.test.sort_wsdl:main
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - pytz
+        - lxml
+        - zope.interface
+        - sqlalchemy
+        - pyzmq
+        - werkzeug
+        - pyparsing
+        - twisted
+        - django
+        - pyramid
+        - msgpack-python
+        - pyyaml
+        - simplejson
+
+test:
+    imports:
+        - spyne
+        - spyne.auxproc
+        - spyne.client
+        - spyne.client.twisted  # [not py3k]
+        - spyne.const
+        - spyne.interface
+        - spyne.interface.wsdl
+        - spyne.interface.xml_schema
+        - spyne.model
+        - spyne.model.primitive
+        - spyne.protocol
+        - spyne.protocol.cloth
+        - spyne.protocol.dictdoc
+        - spyne.protocol.html
+        - spyne.protocol.soap
+        - spyne.server
+        - spyne.server.twisted  # [not py3k]
+        - spyne.store
+        - spyne.store.relational
+        - spyne.test
+        - spyne.test.interface
+        - spyne.test.interface.wsdl
+        - spyne.test.interop
+        - spyne.test.interop.server
+        - spyne.test.interop.server.soap11
+        - spyne.test.interop.server.soap12
+        - spyne.test.model
+        - spyne.test.multipython
+        - spyne.test.multipython.model
+        - spyne.test.protocol
+        - spyne.test.transport
+        - spyne.util
+        - spyne.util.oset
+
+about:
+    home: http://spyne.io
+    license: LGPL-2.1
+    summary: A transport and architecture agnostic rpc library that focuses on exposing public services with a well-defined API
+
+extra:
+    recipe-maintainers:
+        - ocefpaf
+        - emiliom
```

Comment:
Ping @emiliom another one that I am adding in conda-forge but will be fetched on ODM2 later.